tests/pipelines/test_execution_utils.py

```diff
@@ -207,7 +207,7 @@ def name(self):
         def environment(self):
             return {"C": "D"}
 
-    with mock.patch("mlflow.pipelines.utils.execution._exec_cmd") as mock_run_in_subprocess:
+    with mock.patch("mlflow.pipelines.utils.execution._run_make") as mock_run_in_subprocess:
```

Comment:
I'd rather assert that the `_exec_cmd` mock is called twice, since we have separate coverage for `_exec_cmd`.